docs/develop/processing.md

```diff
@@ -1,17 +1,59 @@
 # Pre- and post-processing
 
-When you serialize a model, you want to keep the output as possible, containing just the model and its direct dependencies without custom logic or extra frameworks. This is great for keeping file sizes small and deserialization fast, but makes it tough to know what to do with the serialized model.
+An efficiently serialized model is just the model and its direct dependencies. No custom logic or extra frameworks.
 
-One feature that makes Truss different from model serialization frameworks is bundled pre- and post-processing. Most models intended for use in production systems will want to have these functions as part of the Truss.
+This makes deserialization fast, but may make using the model inconvenient.
 
-Add pre-processing and post-processing functions to your Truss by editing `model/model.py` to include the necessary functions.
+Truss bundles your serialized model with pre- and post-processing functions. Use them to:
 
-## Pre-processing
+* Format model inputs and outputs, especially when the model needs something like a Numpy array that isn't JSON-serializable
```

Comment:
i think technically we handle most numpy objects for serialization, but maybe mention xgboost.DMatrix instead which is a more common issue